code/modules/events/blob/blob_structures/factory.dm

```diff
@@ -1,11 +1,13 @@
+GLOBAL_VAR_INIT(spores_active, 0)
```

Comment:
Is it something that can be scoped to SSmobs?

Reply:
Resolved on discord